utils/docker/docker_entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+cd $HOME/self_balancing_robot
+
+catkin config --extend /opt/ros/melodic --cmake-args -DCMAKE_BUILD_TYPE=RelWithDebInfo
+catkin init && catkin build
+
+echo "source $HOME/self_balancing_robot/devel/setup.bash" >> $HOME/.bashrc
+echo "source $HOME/self_balancing_robot/src/utils/scripts/setup.bash" >> $HOME/.bashrc
+source $HOME/.bashrc
+
+clear
+echo "to start pre-configured tmux session with 3 terminals set up for ROS, use 'bash src/tmux_session.sh'"
```

Comment:
Nitpick:
The hard-coded number "3" in this sentence will probably get outdated very quickly, and its easy to forget to change this when making changes to the Docker in general.
Maybe we should remove it and instead just have the sentence like this:
> "to start a pre-configured tmux session which launches the current software, use ...

(Not important! but makes it easier to maintain without removing much content)